crates/algokit_utils/src/transactions/application_call.rs

```diff
@@ -0,0 +1,142 @@
+use super::common::CommonParams;
+use algokit_transact::{Address, BoxReference, OnApplicationComplete, StateSchema};
+
+/// Parameters for application call transactions.
+#[derive(Debug, Clone)]
+pub struct ApplicationCallParams {
+    pub common_params: CommonParams,
+    /// ID of the application being called.
+    ///
+    /// Set this to 0 to indicate an application creation call.
+    pub app_id: u64,
+    /// Defines what additional actions occur with the transaction.
+    pub on_complete: OnApplicationComplete,
+    /// Transaction specific arguments available in the application's
+    /// approval program and clear state program.
+    pub args: Option<Vec<Vec<u8>>>,
+    /// List of accounts in addition to the sender that may be accessed
+    /// from the application's approval program and clear state program.
+    pub account_references: Option<Vec<Address>>,
+    /// List of applications in addition to the application ID that may be called
```

Comment:
```suggestion
    /// List of applications in addition to the current application that may be called
```
I think this is easier to understand